src/conclusion/conclusion.tex

```diff
@@ -2,3 +2,59 @@
 \chapter{Conclusion}\label{ch:conclusion}
 % textidote: ignore end
 
+The game of chess is one that has stood the tests of time, this is partly due to the highly intricate mechanics that
+make chess an unsolved game.
+As chess is a game where all that matters is winning, much effort has through time has gone into experimenting with
+strategies and new inventions.
+The latest step in chess has been computer chess engines which reached beyond human skill with Deep Blue.
+Engines are now widely used for analysis, but the authors of this paper, have found that they also have immense
+potential for teaching chess.
+
+The saying ``Once a student, now the master'' shows that learning from masters of their craft is nothing new, but
```

Comment:
I dont think this has been mentioned in the report, why we either have to delete it from the conclusion or include it somewhere in the report.